main.go

```diff
@@ -121,6 +126,13 @@ func Exec(ctx context.Context, args Args) error {
 	if err := runCommand(cmdArgs); err != nil {
 		return fmt.Errorf("error executing jfrog rt build-docker-create command: %v", err)
 	}
+	
+	if args.RepoURL != "" && args.BranchName != "" && args.CommitSha != "" {
+		cmdArgs = []string{"jfrog", "rt", "build-add-git", args.BuildName, args.BuildNumber}
+		if err := runCommand(cmdArgs); err != nil {
+			return fmt.Errorf("error executing jfrog rt build-add-git command: %v", err)
+		}
+	}
```

Comment:
let’s introduce a new input `PLUGIN_GIT_PATH` and set the default value to `DRONE_WORKSPACE`. The `jfrog rt build-add-git` command should run from inside that path.